### ISSUE TITLE
Badges should (optionally) include the branch name

### ISSUE THREAD

**Issue by nathanaeljones** [badges/shields#344]
As pass/fail results are tied to a specific branch, yet this is not visually reflected in the badge, users can be led to believe a branch is passing when it is not. Avoiding this would require either maintaining edits to readme.md for each branch (dirtying the history), or displaying multiple badges for the active branches.  Text labels can be used, but GitHub does not align text and images, making this a very ugly solution.

TLDR; Can we support `?label=master` and/or `?passing=master - passing&failing=master - failing` querystring options? [AppVeyor and Waffle](http://help.appveyor.com/discussions/suggestions/461-minor-suggestion-for-status-badges-include-branch-name-in-svg) permit customization, and these make it much easier to provide accurate data to visitors.

P.S., happy to do a pull request if this feature is approved.

**Comment by espadrine** [comment]
We mention at the bottom of the website that we allow you to:

> Override the default left-hand-side text with `?label=wow so healthy`.

That way you can `https://img.shields.io/travis/joyent/node/v0.6.svg?label=wow so healty`

![](https://img.shields.io/travis/joyent/node/v0.6.svg?label=wow so healty)

or alternatively, if you insist, `https://img.shields.io/travis/joyent/node/v0.6.svg?label=master`.


![](https://img.shields.io/travis/joyent/node/v0.6.svg?label=master)

**Comment by nathanaeljones** [comment]
I'm not sure how I missed that. Thank you very much!

**Comment by espadrine** [comment]
Glad to help!
